src/components/block-editor/visual-editor.js

```diff
@@ -36,6 +36,13 @@ import EditorHeading from '../editor-heading-slot';
 function MaybeIframe( { children, contentRef, shouldIframe, styles, style } ) {
 	const ref = useMouseMoveTypingReset();
 
+	const { assets } = useSelect( ( select ) => {
+		const settings = select( 'core/block-editor' ).getSettings();
+		return {
+			assets: settings.__unstableResolvedAssets,
+		};
+	}, [] );
+
```

Comment:
This implementation is necessary to allow the injection of the styles into the iframe preview.
It was based on: https://github.com/WordPress/gutenberg/blob/trunk/packages/block-editor/src/components/block-preview/auto.js#L31-L38